requirements_test.txt

```diff
@@ -1,6 +1,7 @@
 coverage
 coveralls
 flake8
+py-cpuinfo
```

Comment:
Including this here because then it allows developers to run setup.py directly, e.g., ``python setup.py build_ext --inplace``.